app/controllers/admin/dashboard_controller.rb

```diff
@@ -4,4 +4,10 @@ class Admin::DashboardController < Admin::BaseController
   def index
     @orders = Order.sort_orders
   end
+
+  def update_status
```

Comment:
I would only suggest renaming the method more explicitly to reference order status.

Reply:
Agreed with @freeheeling  especially since you're calling a method of `update_status' later in this method. Renaming it to something more explicit removes the ambiguity.

Reply:
@freeheeling and @CoopTang agreed! The action is now called `update_order_status`.